pkg/cmd/cli/restore/create.go

```diff
@@ -171,9 +171,15 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 			return err
 		}
 	case o.ScheduleName != "":
-		if _, err := o.client.VeleroV1().Schedules(f.Namespace()).Get(o.ScheduleName, metav1.GetOptions{}); err != nil {
+		selector := fmt.Sprintf("velero.io/schedule-name=%s", o.ScheduleName)
```

Comment:
you can use the `api.ScheduleNameLabel` constant here in place of the hardcoded label key.

Reply:
thanks for letting me know the constant.
thanks again for your review, very useful to learn more about velero